### DISCUSSION TITLE
rename environment

### DISCUSSION THREAD

**Discussion by Geczy** [coollabsio/coolify#1581] [Feature Requests]
(from production > to anything)

**Comment by RayBB** [comment]
@Geczy I'm able to rename the production environment by clicking the gear icon from the "environments" page.

<img width="694" alt="image" src="https://github.com/coollabsio/coolify/assets/921217/913d1f3f-b845-490d-8d0d-58fb231f66ad">


Does this work for you?

**Comment by AlejandroAkbal** [comment]
This is available, please close this issue
